src/assets/icons/Icons.jsx

```diff
@@ -0,0 +1,89 @@
+export function BiCartFill(props) {
+  return (
```

Comment:
suggestion: I would keep the static assets only in assets, not the components

Reply:
In one of the sessions, I saw Sudesh doing it this way. So I was following the same.